code-challenges.test.js

```diff
@@ -49,9 +113,43 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+describe ("capsOnly", () => {
```

Comment:
You don't need duplicate variables. The variables should all be inside the test.